### PR TITLE
added kube version restrictions to rancher chart

### PR DESCRIPTION
**Pull request by paynejacob**

### CODE REVIEW

File: chart/Chart.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ name: rancher
 description: Install Rancher Server to manage Kubernetes clusters across providers.
 version: %VERSION%
 appVersion: %APP_VERSION%
+kubeVersion: < 1.20.0
```
Comment (cbron) [inline chart/Chart.yaml:6]: How are we going to keep track of this over time ? Question for wider team.

Comment (deniseschannon) [reply chart/Chart.yaml:6]: As part of the introducing new minor versions into Rancher, we should add this as a step.

Comment (deniseschannon) [reply chart/Chart.yaml:6]: @StrongMonkey As part of adding k8s 1.20 into Rancher v2.5 line, please update this kubeVersion. Can you also add it into the process docs about introducing new k8s versions. 

@paynejacob Can you please make a PR into the release/v2.5 branch and master branch?

**Review by deniseschannon — APPROVED** [review]

**Review by cbron — APPROVED** [review]

**Comment by PPCM** [comment]
What is the goal of this restriction?
Currently k3s has a kubversion of 1.20 and it is not possible to install Rancher on fresh install since this morning
We have to downgrade k3s version to 1.19.5

**Comment by deniseschannon** [comment]
It's exactly what you pointed out. Rancher doesn't support k8s 1.20 which is why we have added the k8s version restriction to the helm chart.

**Comment by PPCM** [comment]
OK thanks, I didn't see explanation in the release note, I only saw the restriction without reason

**Comment by guemidiborhane** [comment]
will rancher support k8s >= 1.20 someday?

**Comment by superseb** [comment]
It is tracked in https://github.com/rancher/rancher/issues/30134

**Comment by kubealex** [comment]
what is exactly tracking aside from some tests that should be made on azuread? I mean, can we help in any way to move forward?

**Comment by cbron** [comment]
@kubealex #30134 is tracking adding Rancher support for Kubernetes 1.20. If you have a problem with the K8s version restrictions on the helm chart make sure you are on Rancher v2.5.5 and that the K8s version is <=1.19.x

**Pull request merged**